index.html

```diff
@@ -7,6 +7,7 @@
             content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no"
         />
         <title>Movie APP</title>
+        <link rel="stylesheet" href="./src/styles/styles.css">
```

Comment:
webpack does it all by itself. And we don't have to do it manually.

Reply:
Thank you for response. But now I am only designing the task. That is why I did these changes. I just pulled request to wrong master branch. I should have done it to mine, instead of yours.